src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1696,8 +1696,6 @@ namespace ReportSizingManager {
 							}
 							SupFanNum = PrimaryAirSystem( CurSysNum ).SupFanNum;
```

Comment:
note that for Fan:SystemModel, the indexing is zero-based, but of course for the old fans the indexing is 1-based.